packages/schema/bind/src/bindings/rust/wasm/templates/module-type/module-rs.mustache

```diff
@@ -0,0 +1,61 @@
+{{#moduleType}}
+use crate::{
+    {{#methods}}
+    Args{{#toUpper}}{{name}}{{/toUpper}},
+    {{/methods}}
+};
+use polywrap_wasm_rs::{
+    BigInt,
+    BigNumber,
+    Map,
+    JSON,
+};
+{{#propertyDeps.length}}
+
+{{#propertyDeps}}
+{{^isEnum}}
+use {{crate}}::{{#detectKeyword}}{{#toUpper}}{{type}}{{/toUpper}}{{/detectKeyword}};
+{{/isEnum}}
+{{#isEnum}}
+use crate::{
+    {{#detectKeyword}}{{#toUpper}}{{type}}{{/toUpper}}{{/detectKeyword}},
+};
+{{/isEnum}}
+{{/propertyDeps}}
+{{/propertyDeps.length}}
+
+pub struct Module {
+  {{#hasEnv}}pub env: Option<Env>,{{/hasEnv}}
+}
+{{#hasEnv}}
+
+pub trait EnvTrait {
+  fn __set_env__(&mut self, env: Env) -> ();
+}
+{{/hasEnv}}
+
+pub trait IModule {
```

Comment:
I think we should find a better name for the IModule trait, prefixing it with `I` is not very Rust idiomatic

Reply:
ModuleTrait?